CanalTP/NmmPortalBundle
# Delete client NMM

# Description

This PR add delete client fonction

## JIRA

https://jira.canaltp.fr/browse/BOT-434

## How to test

Here are the following steps to test this pull request:

- login with NMM admin account
- go to client list view 
- you should see "supprimer" button in the dropdown-menu of each client
...

## PR related

https://github.com/CanalTP/SamCoreBundle/pull/47
https://github.com/CanalTP/SamEcoreSecurityManagerBundle/pull/8
